src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -71,6 +60,25 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
         mapboxgl.accessToken = props.config.mapboxToken;
     }
 
+    public shouldComponentUpdate(nextProps: IGeoChartRendererProps) {
+        const {
+            config: { center: nextCenter, zoom: nextZoom },
+        } = nextProps;
+        const { chart } = this;
+
+        if (chart) {
+            const chartCenter = chart.getCenter();
+            const chartZoom = chart.getZoom();
+            return !isEqual(chartCenter, nextCenter) || !isEqual(chartZoom, nextZoom);
+        }
```

Comment:
do we need this?
IMO, Comparing currentCenter with nextCenter, currentZoom with nextZoom are enough